### PR TITLE
*: release v0.9.0

### PR DESCRIPTION
**Pull request by estroz**
**Description of the change:** change versions to `v0.9.0`.

**Motivation for the change:** release `v0.9.0`.

### CODE REVIEW

File: internal/pkg/scaffold/gopkgtoml.go [hunk 1]
```diff
@@ -83,8 +83,8 @@ required = [
 [[constraint]]
   name = "github.com/operator-framework/operator-sdk"
   # The version rule is used for a specific release and the master branch for in between releases.
-  branch = "master" #osdk_branch_annotation
-  # version = "=v0.8.0" #osdk_version_annotation
+	# branch = "master" #osdk_branch_annotation
```
Comment (AlexNPavel) [inline internal/pkg/scaffold/gopkgtoml.go]:
```suggestion
  # branch = "master" #osdk_branch_annotation
```

File: internal/pkg/scaffold/gopkgtoml_test.go [hunk 2]
```diff
@@ -81,8 +81,8 @@ required = [
 [[constraint]]
   name = "github.com/operator-framework/operator-sdk"
   # The version rule is used for a specific release and the master branch for in between releases.
-  branch = "master" #osdk_branch_annotation
-  # version = "=v0.8.0" #osdk_version_annotation
+	# branch = "master" #osdk_branch_annotation
```
Comment (AlexNPavel) [inline internal/pkg/scaffold/gopkgtoml_test.go]:
```suggestion
  # branch = "master" #osdk_branch_annotation
```

File: internal/pkg/scaffold/helm/gopkgtoml.go [hunk 3]
```diff
@@ -38,8 +38,8 @@ func (s *GopkgToml) GetInput() (input.Input, error) {
 const gopkgTomlTmpl = `[[constraint]]
   name = "github.com/operator-framework/operator-sdk"
   # The version rule is used for a specific release and the master branch for in between releases.
-  branch = "master" #osdk_branch_annotation
-  # version = "=v0.8.0" #osdk_version_annotation
+	# branch = "master" #osdk_branch_annotation
```
Comment (AlexNPavel) [inline internal/pkg/scaffold/helm/gopkgtoml.go]:
```suggestion
  # branch = "master" #osdk_branch_annotation
```

File: internal/pkg/scaffold/gopkgtoml.go (hunk 1 above)
Comment (estroz) [reply internal/pkg/scaffold/gopkgtoml.go]: Oops

File: internal/pkg/scaffold/gopkgtoml_test.go [hunk 4]
```diff
@@ -91,5 +91,5 @@ required = [
   [[prune.project]]
     name = "k8s.io/kube-state-metrics"
     unused-packages = true
-  
+
```
Comment (LiliC) [inline internal/pkg/scaffold/gopkgtoml_test.go]: Any reason for this?

File: internal/pkg/scaffold/go_mod_test.go [hunk 5]
```diff
@@ -89,4 +89,6 @@ replace (
 	sigs.k8s.io/controller-tools => sigs.k8s.io/controller-tools v0.1.11-0.20190411181648-9d55346c2bde
 	k8s.io/kube-state-metrics => k8s.io/kube-state-metrics v1.6.0
 )
+
+replace github.com/operator-framework/operator-sdk => github.com/operator-framework/operator-sdk v0.9.0
```
Comment (LiliC) [inline internal/pkg/scaffold/go_mod_test.go]: Why is this now needed but not before?

Comment (estroz) [reply internal/pkg/scaffold/go_mod_test.go]: We [pin](https://github.com/operator-framework/operator-sdk/blob/master/doc/dev/release.md#1-create-a-pr-for-release-version-and-changelogmd-updates) the exact SDK version for releases.

File: internal/pkg/scaffold/gopkgtoml_test.go (hunk 4 above)
Comment (estroz) [reply internal/pkg/scaffold/gopkgtoml_test.go]: My text editor removes extraneous whitespaces. I can revert this but it shouldn't be there to begin with.

**Pull request merged**